docker-compose.yaml

```diff
@@ -0,0 +1,45 @@
+version: '3.4'
+
+x-common-config: &common_config
+  volumes:
+    - home:/home
+  environment:
+    - SQUAD_CELERY_BROKER_URL=amqp://rabbitmq
+    - XDG_DATA_HOME=/home
+
+volumes:
+  home:
+
+services:
```

Comment:
It would be helpful to add "container_name:" to each service so they're easier to debug/identify when running

Reply:
doesn't docker-compose already give nice names to the containers? for me it gives:

```
NAMES
squad_rabbitmq_1
squad_listener_1
squad_web_1
squad_worker_1
squad_scheduler_1
```

Reply:
In my case all of the containers are named "squad".

Reply:
scratch that. I looked at wrong column. Container names are correct